spark/src/test/scala/org/apache/comet/CometCastSuite.scala

```diff
@@ -900,9 +900,7 @@ class CometCastSuite extends CometTestBase with AdaptiveSparkPlanHelper {
   }
 
   test("cast TimestampType to LongType") {
-    assume(
-      !CometConf.isExperimentalNativeScan
-    ) // https://github.com/apache/datafusion-comet/issues/1441
+    // https://github.com/apache/datafusion-comet/issues/1441assume(!CometConf.isExperimentalNativeScan)
```

Comment:
`assume(!CometConf.isExperimentalNativeScan)` got appended to the end of the issue link.

Reply:
This was fixed already in https://github.com/apache/datafusion-comet/pull/1479/commits/61a307148fedf643b39acd27af5fe560fd5a65e6

Reply:
Sorry about that--I must have followed a notification to an old commit.